README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Testing
 
-Pull dependencies
+Requires the [`mops`](https://docs.mops.one/quick-start) package manager for Motoko.
 
 ```
 git submodule update --init
```

Comment:
As the .gitmodules file was deleted: should this be removed from the README?

Reply:
This is addressed in the next PR in the stack https://github.com/dfinity/motoko-bitcoin/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L8.